src/overset/TiogaBlock.C

```diff
@@ -274,35 +278,57 @@ void TiogaBlock::adjust_resolutions()
   constexpr double large_volume = std::numeric_limits<double>::max();
   stk::mesh::Selector mesh_selector = get_node_selector(ovsetParts_);
   const stk::mesh::BucketVector& mbkts = bulk_.get_buckets(
-    meta_.side_rank(), mesh_selector);
+    stk::topology::NODE_RANK, mesh_selector);
+  auto* nodeVol = meta_.get_field<ScalarFieldType>(
+    stk::topology::NODE_RANK, "tioga_nodal_volume");
 
   auto& eidmap = bdata_.eid_map_.h_view;
   auto& cellres = bdata_.cell_res_.h_view;
-  auto& noderes = bdata_.node_res_.h_view;
+
   for (auto b: mbkts) {
-    for (size_t fi=0; fi < b->size(); ++fi) {
-      const auto face = (*b)[fi];
-      const auto* elems = bulk_.begin_elements(face);
-      const auto num_elems = bulk_.num_elements(face);
+    for (size_t in =0; in < b->size(); ++in) {
```

Comment:
You could do this here:
for(auto node : *b) {
if you don't need the index 'in' for anything else.

Reply:
Hold the presses! @alanw0 recommending `auto`??? Addressed in 1ae8ce3

Reply:
Arrggh! This would be better: 'for(stk::mesh::Entity node : *b) {'